### PR TITLE
[tests] Allow to overwrite repository for testing script download

### PR DESCRIPTION
**Pull request by jstourac**
This unifies the path prefix from where the testing scripts are downloaded into one environment property that can be easily overriden - this may be useful during the debugging of the tests execution or implementation.

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,6 +5,7 @@ DATE 			 ?= $(shell date +'%Y%m%d')
 IMAGE_TAG		 ?= $(RELEASE)_$(DATE)
 KUBECTL_BIN      ?= bin/kubectl
 KUBECTL_VERSION  ?= v1.23.11
+NOTEBOOK_REPO_BRANCH_BASE="https://raw.githubusercontent.com/opendatahub-io/notebooks/main"
```
Comment (harshad16) [inline Makefile:8]:
```suggestion
NOTEBOOK_REPO_BRANCH_BASE ?= "https://raw.githubusercontent.com/opendatahub-io/notebooks/main"
```

lets make it generic, so other can run the test from their fork , if they want.

**Review by harshad16 — COMMENTED** [review]
lgtm
might need a small change
https://www.gnu.org/software/make/manual/html_node/Setting.html

File: Makefile (hunk 1 above)
Comment (jstourac) [reply Makefile:8]: Oh, yeah, good catch! I wrote it the `bash` way :upside_down_face: . But it worked as expected too :smile: 

Anyway, I'll update. Thank you :slightly_smiling_face:

**Review by atheo89 — APPROVED** [review]
Good catch Jan! Thank you :slightly_smiling_face:  
/lgtm

**Pull request merged**